VideoLocker/res/layout/row_discussion_thread.xml

```diff
@@ -23,67 +27,63 @@
 
         <TextView
             android:id="@+id/discussion_post_title"
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            android:ellipsize="end"
-            android:maxLines="2"
-            android:textColor="@color/edx_grayscale_neutral_x_dark"
-            android:textSize="@dimen/edx_small"
-            tools:text="This is a post title. It is a very long title to demonstrate that post titles will be truncated when they are too long." />
+            style="@style/discussion_title_text"
+            android:gravity="center_vertical"
+            tools:text="This is a post title. It is a very long title to demonstrate that post titles will not be truncated when they are too long." />
 
         <LinearLayout
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
             android:layout_marginTop="@dimen/x_small_icon_margin"
-            android:gravity="center_vertical"
+            android:gravity="top"
             android:orientation="horizontal">
 
             <org.edx.mobile.view.custom.IconImageViewXml
                 android:id="@+id/discussion_post_closed_icon"
-                android:layout_width="@dimen/edx_x_small"
-                android:layout_height="@dimen/edx_x_small"
-                android:layout_marginEnd="@dimen/discussion_posts_icon_margin"
-                android:layout_marginRight="@dimen/discussion_posts_icon_margin"
-                app:iconColor="@color/edx_grayscale_neutral_base"
+                style="@style/discussion_thread_icon"
+                app:iconColor="@color/edx_grayscale_neutral_dark"
```

Comment:
Better to keep it as is, cuz the layout editor doesn't show the color, if I define it in the style.